pages/myinfo/bookmark/bookmarkType.ts

```diff
@@ -0,0 +1,6 @@
+export interface GetMyinfoBookmarkProps {
+  techArticleId: string;
+  bookmarkSort: 'BOOKMARKED' | 'LATEST' | 'MOST_COMMENTED';
+}
+
+export type MyinfoBookmarkDropdownProps = 'BOOKMARKED' | 'LATEST' | 'MOST_COMMENTED';
```

Comment:
`GetMyinfoBookmarkProps` 는 아래와 같이 작성할 수 있을 것 같아용!

```suggestion
export interface GetMyinfoBookmarkProps {
  techArticleId: string;
  bookmarkSort: MyinfoBookmarkDropdownProps;
}

export type MyinfoBookmarkDropdownProps = 'BOOKMARKED' | 'LATEST' | 'MOST_COMMENTED';
```

Reply:
오옹 감사합니당 !! [6df605e](https://github.com/dreamyPatisiel/devdevdev-client/pull/52/commits/6df605e227f6307143dd746d96cdb4b65effce03)